### PR TITLE
EVAKA-4190: Messaging Playwright Tests

### PR DESCRIPTION
**Pull request by mapehe**
#### Summary

Add an e2e to test sending and replying to a message.

### CODE REVIEW

File: frontend/src/e2e-playwright/pages/citizen/citizen-messages.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  constructor(private readonly page: Page) {}
+
+  #messageReplyContent = new RawTextInput(
+    this.page,
+    '[data-qa="message-reply-content"]'
+  )
+  #threadListItem = new RawElement(this.page, '[data-qa="thread-list-item"]')
+  #sendReplyButton = new RawElement(this.page, '[data-qa="message-send-btn"]')
+
+  async replyToThread(index: number, content: string) {
```
Comment (jaahon) [inline frontend/src/e2e-playwright/pages/citizen/citizen-messages.ts:14]: `index` is an unused parameter.

File: frontend/src/e2e-playwright/pages/citizen/citizen-messages.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    await this.#threadListItem.click()
+    await this.#messageReplyContent.fill(content)
+    await this.#sendReplyButton.click()
+    await this.page.waitForTimeout(1000)
```
Comment (jaahon) [inline frontend/src/e2e-playwright/pages/citizen/citizen-messages.ts:18]: In general it's not a good idea to just do a fixed time timeout. If you need to wait for some async action to complete, read the UI for an indication that it completed.

File: frontend/README.md [hunk 3]
```diff
@@ -213,8 +213,39 @@ test('Application main page is visible', async (t) => {
...
+pm2 stop service
+```
+
+Then start the dev server by running the following command in the `service`
+directory:
+
+```sh
+./gradlew bootRunTest
+```
+
```
Comment (jaahon) [inline frontend/README.md:236]: This is actually not required, so you might want to reword this bit.

Comment (mapehe) [reply frontend/README.md:236]: Thanks for the feedback, could you explain this in a bit more detail?

**Comment by mapehe** [comment]
Thanks for the feedback @jaahon, made some changes accordingly.

File: frontend/README.md (hunk 3 above)
Comment (jaahon) [reply frontend/README.md:236]: I mean that running `service` with `./gradlew bootRunTest` is not required to run playwright tests. It's a useful thing sometimes when you don't want to wipe data from your dev environment, but it's not required.

Comment (mapehe) [reply frontend/README.md]: Ok, thanks. Reworded README a bit.

**Review by jaahon — APPROVED** [review]

**Pull request merged**